modules/kubernetes-addons/prometheus/variables.tf

```diff
@@ -56,3 +56,12 @@ variable "manage_via_gitops" {
   default     = false
   description = "Determines if the add-on should be managed via GitOps."
 }
+
+variable "context" {
```

Comment:
I think it's better to keep this name as `irsa_context` or `irsa_lookups` for better readability.